language-server/src/dotty/tools/languageserver/Signatures.scala

```diff
@@ -0,0 +1,143 @@
+package dotty.tools.languageserver
+
+import dotty.tools.dotc.ast.tpd._
+import dotty.tools.dotc.core.Constants.Constant
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Denotations.SingleDenotation
+import dotty.tools.dotc.util.Positions.Position
+import dotty.tools.dotc.core.Types.{ErrorType, MethodType}
+import dotty.tools.dotc.reporting.diagnostic.messages
+
+import org.eclipse.lsp4j.{ParameterInformation, SignatureInformation}
+
+import scala.collection.JavaConverters._
+
+object Signatures {
```

Comment:
Seems like this could be useful in the REPL or even in error messages, so could be in `compiler/` (but `toSignatureInformation` / `toParameterInformation` would have to be kept in this project)

Reply:
I've moved it to `dotty.tools.dotc.util.Signatures`